sklearn/impute/_iterative.py

```diff
@@ -565,9 +565,6 @@ def fit_transform(self, X, y=None):
 
         self.imputation_sequence_ = []
 
-        self._min_value = -np.inf if self.min_value is None else self.min_value
```

Comment:
Be careful, here we are assigning to `_min_value` and not `min_value`. I think that the best is to change the rest of the code to directly use `self.min_value` instead of `self._min_value`.